orderer/consensus/smartbft/configverifier.go

```diff
@@ -99,7 +99,29 @@ func (cbv *ConfigBlockValidator) ValidateConfig(envelope *cb.Envelope) error {
 }
 
 func (cbv *ConfigBlockValidator) checkConsentersMatchPolicy(conf *cb.Config) error {
-	// TODO: check nil pointers everywhere
+	if conf == nil {
+		return fmt.Errorf("empty Config")
+	}
+
+	if conf.ChannelGroup == nil {
+		return fmt.Errorf("empty channel group")
+	}
+
+	if conf.ChannelGroup.Groups == nil || len(conf.ChannelGroup.Groups) == 0 {
+		return fmt.Errorf("no groups in channel group")
+	}
+
+	if conf.ChannelGroup.Groups["Orderer"] == nil {
+		return fmt.Errorf("no 'Orderer' group in channel groups")
+	}
+
+	if conf.ChannelGroup.Groups["Orderer"].Values == nil {
+		return fmt.Errorf("no values in 'Orderer' group")
+	}
+
+	if conf.ChannelGroup.Groups["Orderer"].Values == nil || len(conf.ChannelGroup.Groups["Orderer"].Values) == 0 {
```

Comment:
This is a duplicate of the line above.  Did you mean:
`if conf.ChannelGroup.Groups["Orderer"].Values["Orderers"] == nil ... `